configure.ac

```diff
@@ -104,54 +139,88 @@ if test "X$HAVE_FORTRAN_2003" = "Xyes"; then
     echo "no"
   else
     echo "yes"
-  fi 
+  fi
+else
+  echo "no"
 fi
 # End check if we have Fortran 2003 and Intel compiler.
 
 AM_CONDITIONAL([FORTRAN_2003_CONDITIONAL_F], [test "X$HAVE_FORTRAN_2003" = "Xyes"])
 AM_CONDITIONAL([FORTRAN_2003_NOTINTEL_CONDITIONAL_F], [test "X$HAVE_FORTRAN_2003_NOTINTEL" = "Xyes"])
+HDF5_GE112="no"
+if version_gt "$H5_LIBVER" "1.11"; then
+  HDF5_GE112="yes"
+fi
+AM_CONDITIONAL([HDF5_GE112], [test "X$HDF5_GE112" = "Xyes"])
+
+HDF5_GE110="no"
+if version_gt "$H5_LIBVER" "1.9"; then
+  HDF5_GE110="yes"
+fi
+AM_CONDITIONAL([HDF5_GE110], [test "X$HDF5_GE110" = "Xyes"])
+AC_SUBST([HDF5_GE110])
+
+AC_CONFIG_FILES([Makefile
```

Comment:
What about java, looks like it never was added?

Reply:
Correct, it was never added.